change/@fluentui-react-combobox-0b29f0f2-48ae-4e62-8463-a1ac7ee83ba7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30539&commitId=34be95f22cbc9c20c7dc77527a32ac626e71523a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Disabled option.active option.chromium.png|1100| Changed|
|Combobox Converged.Open with grouped options.active option.chromium.png|1100| Changed|
|Combobox Converged.Open.active option.chromium.png|1100| Changed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|1100| Changed|
|Combobox Converged.With selection.active option.chromium.png|1100| Changed|
|Combobox Converged.Option with long content.active option.chromium.png|1578| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Disabled option.active option.chromium.png|502| Changed|
|Dropdown Converged.Option with long content.active option.chromium.png|502| Changed|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|502| Changed|
|Dropdown Converged.With selection.active option.chromium.png|502| Changed|
|Dropdown Converged.Open.active option.chromium.png|502| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
These diffs are expected, because of the way the tests were written focus outlines were incorrectly missing from the screenshots. Now with keyborg we have focus outlines appearing correctly